src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header>
+      <ul class="logo-list">
+        <li class="logo-item">
+          <a class="logo-link" href="./index.html">
+            <img alt="logo" src="./images/logo.png"/>
```

Comment:
The `alt` attribute for the logo image should describe the image content as per the checklist. Instead of `logo`, a more descriptive alt text like `Company Logo` would be better.